core/src/main/java/com/homeaway/streamplatform/streamregistry/db/dao/impl/KafkaManagerImpl.java

```diff
@@ -15,11 +15,7 @@
  */
 package com.homeaway.streamplatform.streamregistry.db.dao.impl;
 
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Properties;
+import java.util.*;
```

Comment:
No star imports, please

Reply:
Yeah. Fixed in my local version. Looks like my checkstyle wasn't set up correctly. Ran into some problems rebasing. I will fix it and push new changes.